ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/FileReader/JavaParserFileReaderByText.java

```diff
@@ -0,0 +1,42 @@
+package edu.usb.argos.ASTProcessor;
+
+import com.github.javaparser.JavaParser;
+import com.github.javaparser.ParseResult;
+import com.github.javaparser.ParserConfiguration;
+import com.github.javaparser.ast.CompilationUnit;
+import com.github.javaparser.ast.body.ClassOrInterfaceDeclaration;
+import com.github.javaparser.ast.body.TypeDeclaration;
+
+
+public class JavaParserFileReaderByText implements IFileAnalyzer<CompilationUnit, String> {
+    @Override
+    public CompilationUnit read(String text) {
+        JavaParser javaParser = new JavaParser(new ParserConfiguration());
+        ParseResult<CompilationUnit> parseResult = javaParser.parse(text);
+        if (parseResult.isSuccessful() && parseResult.getResult().isPresent()) {
+            CompilationUnit compilationUnit = parseResult.getResult().get();
+            return compilationUnit;
+        } else {
+            throw new RuntimeException("Error to analyze de code");
```

Comment:
this method could throw an exception, so can you add "throws" to the method to indicate that an exception may be thrown by this method

![image](https://github.com/user-attachments/assets/1976449f-dd79-48df-8bd3-7600e8190ad3)

Reply:
Done